llvm/include/llvm/DebugInfo/DWARF/DWARFDebugLine.h

```diff
@@ -240,7 +240,8 @@ class DWARFDebugLine {
 
     /// Returns the index of the row with file/line info for a given address,
     /// or UnknownRowIndex if there is no such row.
```

Comment:
Update this comment to describe the pair (& consider using a struct with named members for better readability)